tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+"""Provide basic test configuration and fixture root."""
+from io import TextIOWrapper
+from pathlib import Path
+
+import pytest
+import requests_mock
+
+
+def pytest_addoption(parser):
+    parser.addoption("--performance", action="store_true", help="Run performance tests")
+
+
+def pytest_collection_modifyitems(config, items):
+    if config.getoption("--performance"):
+        return
+    skip_custom = pytest.mark.skip(reason="Need --performance option to run")
+    for item in items:
+        if "performance" in item.keywords:
+            item.add_marker(skip_custom)
```

Comment:
this makes it so that performance-related tests don't run by default, which makes the evaluation loop quicker